<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookClassroom</class>
 <widget class="QDialog" name="bookClassroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Classroom</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxStream">
      <property name="styleSheet">
       <string notr="true">color: black; 
font: 700 10pt &quot;Arial&quot;;

</string>
      </property>
      <property name="currentText">
       <string>BIT</string>
      </property>
      <item>
       <property name="text">
        <string>BIT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CEI_I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CEI_II</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditsname">
      <property name="placeholderText">
       <string>Enter subject name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdittname">
      <property name="placeholderText">
       <string>Enter teacher's name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMessage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
